markdown.dtx

```diff
@@ -16722,6 +16746,7 @@ larsers.PipeTable = Ct(larsers.table_row * parsers.newline
                             + V("PipeTable")
                             + V("Verbatim")
                             + V("FencedCode")
+                            + V("JekyllData")
```

Comment:
We likely don't want to allow the block anywhere after the beginning of the document, since it makes second-level headers ambiguous.